plugins/background_fetch/configs.cc

```diff
@@ -123,3 +123,29 @@ BgFetchConfig::readConfig(const char *config_file)
 
   return true;
 }
+
+///////////////////////////////////////////////////////////////////////////
+// Check the configuration (either per remap, or global), and decide if
+// this request is allowed to trigger a background fetch.
+//
+bool
+BgFetchConfig::bgFetchAllowed(TSHttpTxn txnp) const
+{
+  TSDebug(PLUGIN_NAME, "Testing: request is internal?");
+  if (TSHttpTxnIsInternal(txnp)) {
+    return false;
+  }
+
+  bool allow_bg_fetch = true;
+
+  // We could do this recursively, but following the linked list is probably more efficient.
```

Comment:
This comment is possibly either TMI or just wrong.  It would be a tail call/recursion that the compiler would likely turn into a jump/loop anyway.  The for loop is better because it's more compact.